sql/sql_insert.cc

```diff
@@ -4739,6 +4739,25 @@ bool select_create::store_values(List<Item> &values)
                                               true, TRG_EVENT_INSERT);
 }
 
+#ifdef WITH_WSREP
```

Comment:
Please move this into `wsrep_mysqld.cc`.

Reply:
Done.